Patches/Vanilla Factions Expanded - Mechanoids/ThingDefs_Races/Races_AdvancedMechanoid_PlayerControlled.xml

```diff
@@ -0,0 +1,634 @@
+      <value>
+        <li Class="CombatExtended.RacePropertiesExtensionCE">
+          <bodyShape>Quadruped</bodyShape>
+        </li>
+      </value>
+    </li>
+
+    <li Class="PatchOperationAdd">
+      <xpath>Defs/ThingDef[defName="VFE_Mech_AdvancedCarpenter"]/statBases</xpath>
+      <value>
+        <CarryWeight>400</CarryWeight>
+        <CarryBulk>80</CarryBulk>
+        <MeleeDodgeChance>0.04</MeleeDodgeChance>
+        <MeleeCritChance>0.47</MeleeCritChance>
+        <MeleeParryChance>0.52</MeleeParryChance>
+        <MaxHitPoints>500</MaxHitPoints>
+      </value>
+    </li>
+
+    <li Class="PatchOperationReplace">
+      <xpath>Defs/ThingDef[defName="VFE_Mech_AdvancedCarpenter"]/statBases/ArmorRating_Blunt</xpath>
+      <value>
+        <ArmorRating_Blunt>48</ArmorRating_Blunt>
+      </value>
+    </li>
+
+    <li Class="PatchOperationReplace">
+      <xpath>Defs/ThingDef[defName="VFE_Mech_AdvancedCarpenter"]/statBases/ArmorRating_Sharp</xpath>
+      <value>
+        <ArmorRating_Sharp>22</ArmorRating_Sharp>
+      </value>
+    </li>
+
+    <li Class="PatchOperationReplace">
+      <xpath>Defs/ThingDef[defName="VFE_Mech_AdvancedCarpenter"]/tools</xpath>
+      <value>
+        <tools>
+          <li Class="CombatExtended.ToolCE">
+            <label>head</label>
+            <capacities>
+              <li>Blunt</li>
+            </capacities>
+            <power>35</power>
+            <cooldownTime>3.51</cooldownTime>
+            <linkedBodyPartsGroup>HeadAttackTool</linkedBodyPartsGroup>
+            <armorPenetrationSharp>0</armorPenetrationSharp>
+            <armorPenetrationBlunt>15</armorPenetrationBlunt>
+            <alwaysTreatAsWeapon>true</alwaysTreatAsWeapon>
+          </li>
+        </tools>
+      </value>
+    </li>-->
+          <!--
+
+    -->
+          <!-- ==========VFE Advanced Termite ========== -->
+
+          <li Class="PatchOperationAddModExtension">
+            <xpath>Defs/ThingDef[defName="VFE_Mech_AdvancedTermite_PlayerControlled"]</xpath>
+            <value>
+              <li Class="CombatExtended.RacePropertiesExtensionCE">
+                <bodyShape>QuadrupedLow</bodyShape>
+              </li>
+            </value>
+          </li>
+
+          <li Class="PatchOperationReplace">
+            <xpath>Defs/ThingDef[defName="VFE_Mech_AdvancedTermite_PlayerControlled"]/statBases</xpath>
+            <value>
+              <statBases>
+                <CarryWeight>300</CarryWeight>
+                <CarryBulk>60</CarryBulk>
+                <AimingAccuracy>1.0</AimingAccuracy>
+                <ShootingAccuracyPawn>1</ShootingAccuracyPawn>
+                <MeleeDodgeChance>0.04</MeleeDodgeChance>
+                <MeleeCritChance>0.16</MeleeCritChance>
+                <MeleeParryChance>0.24</MeleeParryChance>
+                <AimingDelayFactor>1.25</AimingDelayFactor>
+                <MaxHitPoints>400</MaxHitPoints>
+                <BandwidthCost>5</BandwidthCost>
```

Comment:
Any specific reason for this node?

Reply:
That patch (except 602 line) for Advanced termit (player controlled) copied from AdvTermit patch.
Line 602 copied from target of patch, cuz its replace-patch

Reply:
Then if the pawn has the StatBases node with stuff inside it, it's better to replace only the necessary bits and add the rest instead of replacing the whole node, we don't want to replace stuff not related to CE.

Reply:
Ok, i'll fix it.

Reply:
Done